Projects/App/Sources/UI/Organization/OrgDetail/OrgDetailViewController.swift

```diff
@@ -0,0 +1,138 @@
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+    }
+
+    // MARK: - Function
+
+}
+
+// MARK: - UI Function
+
+extension OrgDetailViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .zestyColor(.background)
+        
+        orgDetailInformationView1 = OrgDetailInformationView()
+        orgDetailInformationView2 = OrgDetailInformationView()
+        orgDetailInformationView3 = OrgDetailInformationView()
+        
+        orgNameLabel.text = "애플 디벨로퍼 아카데미"
+        orgNameLabel.textColor = .black
+        orgNameLabel.font = UIFont.systemFont(ofSize: CGFloat(26), weight: .bold)
+        orgNameLabel.textAlignment = .center
+        orgNameLabel.numberOfLines = 2
+        
+        orgInformationSuperStackView.axis = .vertical
+        orgInformationSuperStackView.alignment = .top
+        orgInformationSuperStackView.distribution = . fillProportionally
+        orgInformationSuperStackView.spacing = 50
+        
+        orgInformationStackView1.axis = .horizontal
+        orgInformationStackView1.alignment = .center
+        orgInformationStackView1.distribution = .fillEqually
+        orgInformationStackView1.spacing = 30
+        
+        orgInformationStackView2.axis = .horizontal
+        orgInformationStackView2.alignment = .center
+        orgInformationStackView2.distribution = .fillEqually
+        orgInformationStackView2.spacing = 30
+
+        orgInformationStackView3.axis = .horizontal
+        orgInformationStackView3.alignment = .center
+        orgInformationStackView3.distribution = .fillEqually
+        orgInformationStackView3.spacing = 30
+
+        orgInformationImage1.image = UIImage(.img_reviewfriends_together)
+        orgInformationImage1.contentMode = .scaleAspectFit
+        orgDetailInformationView1.informationLabel.text = "함께하는 친구들"
+        orgDetailInformationView1.numberLabel.text = "13,966명"
+        
+        orgInformationImage1.image = UIImage(.img_categoryfriends_western)
+        orgInformationImage1.contentMode = .scaleAspectFit
+        orgDetailInformationView2.informationLabel.text = "등록된 맛집"
+        orgDetailInformationView2.numberLabel.text = "1,425곳"
+        
+        orgInformationImage3.image = UIImage(.img_reviewfriends_photo)
+        orgInformationImage3.contentMode = .scaleAspectFit
+        orgInformationImage3.layer.applyFigmaShadow(color: .black, opacity: 0.05, xCoord: 0, yCoord: 0, blur: 5, spread: 0)
+        orgDetailInformationView3.informationLabel.text = "업로드된 사진"
+        orgDetailInformationView3.numberLabel.text = "124,513개"
+        
+        orgInviteButton.setTitle("우리학교 사람들 초대하기", for: .normal)
+    }
+
+    private func createLayout() {
+        view.addSubviews([orgNameLabel, orgInformationSuperStackView, orgInformationStackView1, orgInformationStackView2, orgInformationStackView3, orgInviteButton])
```

Comment:
p4;

orgInformationSuperStackView에 하위뷰로 orgInformationStackView들이 추가되기 때문에 view.addSubview에는 들어갈 필요가 없습니다~

```suggestion
        view.addSubviews([
            orgNameLabel,
            orgInformationSuperStackView,
            orgInviteButton
        ])
```

Reply:
[d1f354a](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/70/commits/d1f354ad3efc3349b294ec018a2c4a204eacdd70) 하나 배웠습니다 굿굿